docs/draft/howto/consuming-metrics.md

```diff
@@ -253,7 +282,7 @@ metadata:
 spec:
   endpoints:
     - path: /metrics
-      port: https
+      port: metrics
```

Comment:
We need to ensure that is == https://github.com/openshift/operator-framework-operator-controller/blob/0b44e8ce155a7c32660eeff6d862025c9d015880/openshift/catalogd/manifests/17-servicemonitor-openshift-catalogd-catalogd-metrics-monitor.yml#L11-L22

So 👍